README.md

```diff
@@ -1,2 +1,64 @@
-# harry-potter-quarkus
-Showcase in a web application Quarkus and Infinispan
+# Quarkus demo: Infinispan Client
+
+This example showcases how to use Infinispan client with Quarkus.
+* Create stores dynamically
+* Simple get/put operations
+* Full-text query
+* Continuous query
+
+Uses quarkus features, such as
+* REST web-service
+* Web Socket
+* Task Scheduling
+* Application properties
+* Application lifecycle events listeners
+
+# Run infinispan server
+
+- Running with docker `docker run -it -p 11222:11222 jboss/infinispan-server:latest`
+- Download the server from `http://www.infinispan.org` and run `./bin/standalone.sh`
+
+# Harry-Potter demo
+
+The demo will load characters and spells. Characters can execute spells, and we want to monitor who is in
+Hogwarts and performing magic.
+
+A Character has an id, name, biography and type (other, student, teacher, muggle).
+A Spell has an id, name and description.
+A Spell is performed by a character. This character may or may not be at Hogwarts.
```

Comment:
I would use backticks for the class names.

Reply:
ok